### PR TITLE
Decreasing the length of the text and improving consistency of the narrative: `03.use-cases-c.md`

### PR DESCRIPTION
**Pull request by pedrohbraga**
Hi, again

I have decreased the length of this section and elaborated on the context of backwards / version compatibility along with the automation that runners on GitHub Actions provide.

Please do not hesitate to let me know if there are issues or other suggestions here.

### CODE REVIEW

File: content/05.use-cases-c.md [hunk 1]
```diff
@@ -2,66 +2,58 @@
...
 
-Conventional research practices typically rely on one or two people running and checking the data analyses, while most coauthors (and readers of the subsequent publication) see only the final results and a verbal description of the analytical steps.
-In the developmental stages, collaborators can directly see the code for the analysis, manipulate and explore the data themselves, and check for errors.
-Cynically, there is also more insurance against nefarious colleagues that may be tempted to distort results [@url:https://ecologyforthemasses.com/2020/02/04/pruittdata-and-the-ethics-of-data-in-science].
-Collaborators as well as pre-publication and post-publication reviewers are better positioned to discover questionable findings if they have full and transparent access to the project.
+Conventional research practices typically separate tasks among collaborators (_i.e._, data entering, analyses, writing).
+It is not rare that coauthors discuss, but do not actively verify, edit or execute research tasks that are not their main responsibility.
```
Comment (DrMattG) [inline content/05.use-cases-c.md:12]: "It is common..." rather than "it is not rare..."?

File: content/05.use-cases-c.md [hunk 2]
```diff
@@ -2,66 +2,58 @@
...
-Additionally, GitHub allows for repositories and organizations to have designated Code Owners [@url:https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners]; these code owners can change through time allowing for the transition of code from one cohort of researchers to the next [@doi:10.1890/ES14-00402.1].
-There are also multiple means by which to archive code dependency information with GitHub.
-Dependencies can be described in README files, added to environment.yml and/or an environment.txt file, or stored in a repository using tools such as renv (<https://rstudio.github.io/renv/articles/renv.html>).
-This ensures that as packages continue to develop with new syntax, code will still run as it did when it was developed.
+On one hand, GitHub can facilitate project continuity among research code by making code handover to succeding researchers easier [see @doi:10.5334/jors.307; @doi:10.1186/1751-0473-8-7].
```
Comment (DrMattG) [inline content/05.use-cases-c.md:30]: I think that the "on one hand" and later the "on the other hand" can be deleted without altering the message here

File: content/05.use-cases-c.md [hunk 3]
```diff
@@ -2,66 +2,58 @@
...
-This ensures that as packages continue to develop with new syntax, code will still run as it did when it was developed.
+On one hand, GitHub can facilitate project continuity among research code by making code handover to succeding researchers easier [see @doi:10.5334/jors.307; @doi:10.1186/1751-0473-8-7]. 
+Through its version control, the history of code and data from projects in ecology and evolution can be tracked and be accessible to future laboratory members and collaborators [@doi:10.1038/s41559-017-0160].
+Repositories and organizations can have designated data and code owners [or more appropriate, stewards; see @doi:10.1890/ES14-00402.1] [@url:https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners], who can also can change through time allowing for the transition of code from one cohort of researchers to the next.
+
+On the other hand, GitHub allows for informantion and dependencies on software version backwards compatibility to be described and store, to ensure that code will appropriately run as it did when it was developed.
```
Comment (DrMattG) [inline content/05.use-cases-c.md:34]: "GitHub allows for information". 

Is "software version backwards compatibility" correct? It is hard to read - but I guess technically correct.

File: content/05.use-cases-c.md [hunk 4]
```diff
@@ -2,66 +2,58 @@
...
+This can be achieved through the access of GitHub repositories containing the necessary releases of such software, or through the usage of runners and actions with specific software versions in its automation tool, GitHub Actions.
```
Comment (DrMattG) [inline content/05.use-cases-c.md:35]: What are "runners"? - should this be in the glossary box?

File: content/05.use-cases-c.md [hunk 5]
```diff
@@ -2,66 +2,58 @@
...
 In (Figure @fig:scatterblob), we highlight several elements of good repository structure, and the various ways that contributors may interact with them. 
 
 ### GitHub Organizations
 
-<!--*Contributors to this section: Katherine Hébert, Cole Brookson*-->
+<!--*Contributors to this section: Katherine Hébert, Cole Brookson, PHPB*-->
 
-GitHub Organizations offer a shared virtual space that allows a team to work in different repositories, while remaining tied together under a larger figurehead, such as a laboratory, a department, an organization, or a large project involving several teams. 
-Organizations are well-suited to ensure larger projects with many steps or moving parts are constrained to one virtual space, where outputs and sub-projects can be easily accessed and located without relying on any one individual. 
+GitHub Organizations shared virtual spaces that allow teams to work in different repositories, while remaining tied together under a larger figurehead, such as a laboratory, a department, an organization, or a large project involving several teams.
```
Comment (DrMattG) [inline content/05.use-cases-c.md:46]: "GitHub Organizations shared virtual spaces allow teams..."

**Review by DrMattG — APPROVED** [review]
A few comments from me... looks great in general!

File: content/05.use-cases-c.md (hunk 2 above)
Comment (robcrystalornelas) [reply content/05.use-cases-c.md:30]: Agreed, updated text.

File: content/05.use-cases-c.md (hunk 3 above)
Comment (robcrystalornelas) [reply content/05.use-cases-c.md:34]: I think that sentence needs to be reworded. I gave a try at revising.

File: content/05.use-cases-c.md (hunk 4 above)
Comment (robcrystalornelas) [reply content/05.use-cases-c.md:35]: I think it's too technical and not necessary to goal of sentence so will remove.

**Review by robcrystalornelas — APPROVED** [review]
Addressed edits suggested by @DrMattG and made my own edits to make some sections more concise.

**Pull request merged**